src/components/program-progress/ProgramProgressSidebar.jsx

```diff
@@ -32,9 +32,9 @@ const ProgramProgressSideBar = () => {
     [programData],
   );
   return (
-    <div className="program-sidebar col-lg-4">
+    <div className="program-sidebar offset-1 col-3">
       {!programCertificate && <ProgramProgressCircle /> }
-      {programCertificate && (
+      {programCertificate && programCertificate?.img && (
```

Comment:
nit: We can remove `?` from second condition. Also, what will be shown now in place of program certificate when the type is other than `Micromasters`, `XSeries`, and `Professional Certificate`?

Reply:
We will not show anything as it is being handled on B2C side.

Reply:
So how will the learner be able to access the program certificate then?